version.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "1.8.5.2",
+  "publicReleaseRefSpec": [
+    "^refs/heads/master$", // we release out of master
```

Comment:
afaik json doesn't support comments at all

Reply:
It does with .NET's parsers.